pkg/modules/libreoffice/unoconv/unoconv.go

```diff
@@ -243,9 +244,11 @@ func (mod Unoconv) Extensions() []string {
 		".swf",
 		".sxd",
 		".sxw",
+		".tif",
```

Comment:
I didn't find `.tif` in the list of supported file extensions (`unoconv --show`) 🤔  `.tif` == `.tiff` ?

Reply:
Yes you are right. But unoconv actually supports .tif.

Reply:
jpeg isn't in the list too, but is supported by unoconv btw.